packages/vue-cli-plugin-typescript/generator/index.js

```diff
@@ -12,11 +12,15 @@ module.exports = (api, options, rootOptions) => {
       '@coveo/headless': '*',
       '@coveo/search-token-server': '*',
       buefy: '^0.9.4',
+      bulma: '^0.9.3',
```

Comment:
Even if [Buefy](https://buefy.org/) depends on [Bulma](https://bulma.io/), it has to be specified in the dependencies. Otherwise, the module is not found (when installed with pnpm).